app/views/devise/passwords/new.html.erb

```diff
@@ -1,16 +1,26 @@
-<h2>Forgot your password?</h2>
+<%= image_tag 'background.jpg', class: 'background-image' %>
 
-<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+<div class="row container">
+  <div class="col s12 m9 l3">
+    <div class="card sign-in-modal">
+      <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+      <div class="card-content">
+        <span class="card-title">Forgot your password?</span>
+        <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+        <div class="field">
+          <%= f.label :email %><br />
+          <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+        </div>
+      </div>
+      <div class="card-action">
+        <div>
+          <%= render "devise/shared/links" %>
+        </div>
+        <br>
+        <%= f.submit "Send me reset password instructions", class: "btn-large waves-effect waves-light" %>
```

Comment:
The button is too big.  Change the text to "Email Me", then add a little paragraph below the email field explaining that a link will be sent to their email.